### DISCUSSION TITLE
New LoRA: Baize

### DISCUSSION THREAD

**Discussion by oobabooga** [oobabooga/text-generation-webui#758] [General]
> Baize is an open-source chat model fine-tuned with [LoRA](https://github.com/microsoft/LoRA). It uses 100k dialogs generated by letting ChatGPT chat with itself. We also use Alpaca's data to improve its performance. We have released 7B, 13B and 30B models. 60B model coming soon. Please refer to the [paper](https://arxiv.org/pdf/2304.01196.pdf) for more details.

https://github.com/project-baize/baize

**Comment by QuantumAlignmentLabOnline** [comment]
What's it good at?

**Reply by oobabooga** [reply]
The paper has some examples https://arxiv.org/pdf/2304.01196.pdf
